lib/internal/modules/esm/loader.js

```diff
@@ -179,39 +178,37 @@ function nextHookFactory(chain, meta, { validateArgs, validateOutput }) {
  * the main module and everything in its dependency graph.
  */
 class ESMLoader {
-  /**
-   * Prior to ESM loading. These are called once before any modules are started.
-   * @private
-   * @property {KeyedHook[]} globalPreloaders Last-in-first-out
-   *  list of preload hooks.
-   */
-  #globalPreloaders = [];
-
-  /**
-   * Phase 2 of 2 in ESM loading.
-   * @private
-   * @property {KeyedHook[]} loaders Last-in-first-out
-   *  collection of loader hooks.
-   */
-  #loaders = [
-    {
-      fn: defaultLoad,
-      url: 'node:internal/modules/esm/load',
-    },
-  ];
-
-  /**
-   * Phase 1 of 2 in ESM loading.
-   * @private
-   * @property {KeyedHook[]} resolvers Last-in-first-out
-   *  collection of resolver hooks.
-   */
-  #resolvers = [
-    {
-      fn: defaultResolve,
-      url: 'node:internal/modules/esm/resolve',
-    },
-  ];
+  #hooks = {
+    __proto__: null,
+    /**
+     * Prior to ESM loading. These are called once before any modules are started.
+     * @private
+     * @property {KeyedHook[]} globalPreload Last-in-first-out list of preload hooks.
+     */
+    globalPreload: [],
+    /**
+     * Phase 2 of 2 in ESM loading.
```

Comment:
```suggestion
     * Phase 2 of 2 in ESM loading. (Phase 1 is below.)
```

Can we also add some empty lines between these properties?